### DISCUSSION TITLE
Support for BO3

### DISCUSSION THREAD

**Discussion by WECoyote99** [Laupetin/OpenAssetTools#80] [Q&A] [answered]
So I like how this is setup so far for BO2, BO1, etc. But in the future is BO3 on the table? As currently can't make separate ff files for mods and that be useful for patching and mods as you can limit the asset limit to what you want on per map.

**Comment by Laupetin** [comment, marked as answer]
Hey :) Unfortunately i don't think it's very likely this will be a thing for the near future at least. The reasons are the following:
* From the 5 games currently supported only two lead the development with the 3 having pretty barebones support at the moment. It just requires a lot of work so it will take some time until all games have similar feature levels. Adding another game currently is probably not a good idea until the existing ones are working well enough.
* All currently supported games are 32bit. While i plan on tackling 64bit games in the future somewhen, it will require a lot of adjusting existing systems for it and will result in a larger change in how the tool works probably. So it will probably take a while until OAT gets to that point.
* BO3 just has pretty decent mod tools. I think there are other titles that would be interesting to tackle first because it would make a greater impact there. Also since the games get more complex the more time progresses it is probably easier to at least work through the time lines and not skip too many titles.

That's my current thoughts on it. Since OAT's goal is to provide modtools itself and not necessarly patch existing ones it might be more reasonable to tackle your problem with something that does not necessarly try to replace the entire mod tools? 😅

**Reply by WECoyote99** [reply]
> Hey :) Unfortunately i don't think it's very likely this will be a thing for the near future at least. The reasons are the following:
> 
> * From the 5 games currently supported only two lead the development with the 3 having pretty barebones support at the moment. It just requires a lot of work so it will take some time until all games have similar feature levels. Adding another game currently is probably not a good idea until the existing ones are working well enough.
> 
> * All currently supported games are 32bit. While i plan on tackling 64bit games in the future somewhen, it will require a lot of adjusting existing systems for it and will result in a larger change in how the tool works probably. So it will probably take a while until OAT gets to that point.
> 
> * BO3 just has pretty decent mod tools. I think there are other titles that would be interesting to tackle first because it would make a greater impact there. Also since the games get more complex the more time progresses it is probably easier to at least work through the time lines and not skip too many titles.
> 
> 
> 
> That's my current thoughts on it. Since OAT's goal is to provide modtools itself and not necessarly patch existing ones it might be more reasonable to tackle your problem with something that does not necessarly try to replace the entire mod tools? 😅 

Ok that is fully understandable. My question is does BO3 have the ability to make shared ff files like BO1 and 2 does? That is what I'm hoping on as I can then reduce my main ff file size so like for example changes for Shadows of Evil only load for that map that's my big hurdle if not there a way to do so. I only saw this as a tool for shared ff files that does not replace the whole modtools
